skt/templates/partials/test_details.j2

```diff
@@ -1,5 +1,6 @@
 
-We booted the compiled kernel and ran the following hardware tests:
+We booted the compiled kernel and ran the following tests on physical
+hardware:
```

Comment:
This is better, thank you, Major.
However, we often run tests in VMs. Would this sentence still make sense to people then?
Also, we have "hardware tests" in other places of the report, IIRC. Could you fix that as well?

Reply:
All I'm saying, is could we call it just "tests"?